src/go/k8s/controllers/redpanda/cluster_controller.go

```diff
@@ -169,12 +176,14 @@ func (r *ClusterReconciler) Reconcile(
 		clusterSvc,
 		nodeportSvc,
 		ingress,
+		resources.NewSuperUsers(r.Client, &redpandaCluster, r.Scheme, headlessSvc.HeadlessServiceFQDN(r.clusterDomain), !setSuperUsersInRedpanda, log),
 		resources.NewConfigMap(r.Client, &redpandaCluster, r.Scheme, headlessSvc.HeadlessServiceFQDN(r.clusterDomain), log),
 		pki,
 		sa,
 		resources.NewClusterRole(r.Client, &redpandaCluster, r.Scheme, log),
 		crb,
 		sts,
+		resources.NewSuperUsers(r.Client, &redpandaCluster, r.Scheme, headlessSvc.HeadlessServiceFQDN(r.clusterDomain), setSuperUsersInRedpanda, log),
```

Comment:
does the order matter or why is one superuser here and one below ingress?

Reply:
It's described https://github.com/vectorizedio/redpanda/pull/1893#discussion_r677422921